.github/workflows/generate-site.yaml

```diff
@@ -1,32 +1,48 @@
 name: "Generate Static Site"
+defaults:
+  run:
+    shell: bash
 on:
   workflow_call:
     inputs:
       repos:
-        required: true
+        required: false
+        type: string
+      owner:
+        required: false
+        type: string
+      name:
+        required: false
```

Comment:
this adds the `owner` and `name` options so that the calling workflow is _much_ simplified

- before: [old and busted](https://github.com/zkamvar/evals-test/blob/5a8acba1de7d1f7783126dcca26a074b588bd45f/.github/workflows/rebuild-data.yaml)
- after: [new hotness](https://github.com/zkamvar/evals-test/blob/743676d4e90ebe4aad0be6fce73799c4ae5a6fa7/.github/workflows/rebuild-site.yaml)

Note by setting everything to be `required: false`, I'm also able to accommodate the old-style workflows.